test/behavior/nesting.ts

```diff
@@ -643,15 +719,17 @@ async function shouldBehaveLikeNesting(
       const toOwner = tokenOwner.address;
       const notOwner = addrs[3];
       await expect(
-        parent.connect(notOwner).unnestChild(parentId, 0, toOwner, true),
+        parent.connect(notOwner).unnestChild(parentId, toOwner, 0, child.address, childId, true),
       ).to.be.revertedWithCustomError(child, 'ERC721NotApprovedOrOwner');
     });
 
-    it('cannot unnest not existing child', async function () {
+    it('cannot unnest from not existing parent', async function () {
```

Comment:
Same question as above. Should we add another example to validate that attempting to unnest a child with invalid index gets reverted as well?